MobiFlight/MidiBoard/MidiBoardDeviceType.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Runtime.Serialization;
+
+namespace MobiFlight
+{
+    public enum MidiBoardDeviceType
+    {
+        [EnumMember(Value = "Button")]
```

Comment:
Is this needed to serialize/deserialize from JSON? I thought it would Just Work(tm) with the strings, no decorations required.

Reply:
Thanks, I'm not sure. I'll check. There are other classes where it was done like that.

Reply:
Works. Thumbs up.